account_invoice_validation_queued/README.rst

```diff
@@ -0,0 +1,122 @@
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--invoicing-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-invoicing/tree/11.0/account_invoice_validation_queued
+    :alt: OCA/account-invoicing
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-invoicing-11-0/account-invoicing-11-0-account_invoice_validation_queued
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/95/11.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows to enqueue in several jobs the account validation process
+to be executed in paralell on background, which is normally done serially and
+on foreground.
+
+Each invoice creates a job for performing the validation, but you will only be
+able to enqueue invoices for the same date.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Installation
+============
+
+This module depends on *queue_job* module that is hosted on
+https://github.com/OCA/queue.
+
+Configuration
+=============
+
+Jobs are enqueued in the channel ``root.account_invoice_validation_queued``,
+so you must adjust your
+`Odoo configuration <https://github.com/OCA/queue/tree/11.0/queue_job#configuration>`_
+according this.
+
+If you want to see queued jobs, you need "Job Queue / Job Queue Manager"
+permission in your user.
+
+Usage
+=====
+
+#. Go to *Invoicing > Sales > Documents > Customer Invoices* or
+   *Invoicing > Purchases > Documents > Vendor Bills*.
+#. Mark at least one check on the left part of one draft invoice line in the
+   list view.
+#. Click on *Action > Confirm Draft Invoice*.
+#. On the dialog popup that appears, click on "Enqueue Validation".
+#. If any of the selected invoices are not in draft state, you will receive
+   an error.
+#. If any of the invoices is already enqueued, there will be a message saying
+   so and avoiding to perform the process.
+#. Once enqueued, and having the "Job Queue Manager" permission, you can go to
+   the invoice, and see the tab "Validation Jobs". A list with all the jobs
+   related to that invoice can be found there.
+
+Known issues / Roadmap
+======================
+
+* Allow to enqueue for validating invoices of different date.
+* There's a chance that if you perform several enqueues of invoice validations
+  from different dates, the order of the validated invoices, and thus, the
+  number given for it, will result disordered.
```

Comment:
I think that you must include as know issue that no gap sequence implementation not allowed due to database locks

Reply:
Great point! I have included in CONFIGURE section, as it's something to be configured for the module.